perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -18,7 +18,7 @@
 set -e
 sudo apt-get update
 
-readonly INTEGRATION_TEST_TIMEOUT=24m
```

Comment:
As discussed: 
1. this is a periodic test, so it's okay to use low-computation (standard-16) power VM.
2. For presubmit test, we will be using standard-32 machines, which took lesser time.

We can increase further to 40m, so that we don't have to change the timeout frequently as we add more tests.

CC: @vadlakondaswetha

Reply:
Increased to 40m.